index.html

```diff
@@ -658,11 +658,13 @@ <h3>Proofs</h3>
           <dd>
 A string value that contains the data necessary to verify the digital proof
 using the `verificationMethod` specified. The contents of the value MUST be a
-[[MULTIBASE]]-encoded binary value. Alternative fields with different encodings MAY
-be used to encode the data necessary to verify the digital proof instead of this
-value. The contents of this value are determined by a cryptosuite and set to the
-<em>proof value</em> generated by the
-<a href="#add-proof">Add Proof Algorithm</a>.
+[[?MULTIBASE]]-encoded binary value with a header and encoding described in
+<a href="#multibase-0"></a>. The contents of this value are determined by a
+specific cryptosuite and set to the <em>proof value</em> generated by the
+<a href="#add-proof">Add Proof Algorithm</a> for that cryptosuite. Alternative
+properties with different encodings specified by the cryptosuite MAY be used,
+instead of this property, to encode the data necessary to verify the digital
+proof.
```

Comment:
```suggestion
[[?MULTIBASE]]-encoded binary value with a header and encoding as described in
<a href="#multibase-0"></a>. The contents of this value are determined by a
specific cryptosuite and set to the <em>proof value</em> generated by the
<a href="#add-proof">Add Proof Algorithm</a> for that cryptosuite. Alternative
properties with different encodings specified by the cryptosuite MAY be used,
instead of this property, to encode the data necessary to verify the digital
proof.
```